examples/jpeg/example_mozjpeg_image.go

```diff
@@ -7,7 +7,7 @@ import (
 	"io/ioutil"
 	"os"
 
-	"github.com/davidbyttow/govips/v2/vips"
```

Comment:
I don't know how to keep the module paths separate. Can you use `replace` in go.mod? What can we do?